snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/rule532a.groovy

```diff
@@ -0,0 +1,88 @@
+package scripts
+
+import static com.b2international.snowowl.snomed.common.SnomedConstants.Concepts.UK_MAINTAINED_CLINICAL_MODULE
```

Comment:
Please make sure no UK content leaks through to this repo. Pro tip: Ctrl + Shift + O will organize the import statements in any Groovy file

Reply:
Done